relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -304,6 +306,12 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         if let Some(status) = trace_context.status.value() {
             tags.insert(SpanTagKey::TraceStatus, status.to_string());
         }
+
+        if let Some(client_sample_rate) = trace_context.client_sample_rate.value() {
+            if *client_sample_rate > 0. {
+                tags.insert(SpanTagKey::ClientSampleRate, client_sample_rate.to_string());
```

Comment:
Should we insert the sample rate as a `span.measurement` instead? That way, it's sent as a number and won't be indexed unnecessarily.

Reply:
sure, where would that logic go? Measurement extraction doesn't have access to the context

Reply:
If you add it in `extract_segment_measurements` it should end up on the span automatically.